File: MoloAirSite/src/app/models/Booking.ts
export class Booking{
    bookingId: Number;
    customerId: Number;
    flightId: Number;
    referenceNumber: String;

    constructor(bookingId: Number, customerId: Number, flightId: Number, referenceNumber: String){
        this.bookingId=bookingId;
        this.customerId = customerId;
        this.flightId = flightId;
        this.referenceNumber = referenceNumber;
    }

}

File: booking-service/src/main/java/entelect/training/incubator/model/Booking.java
package entelect.training.incubator.model;

import lombok.Data;
import lombok.NoArgsConstructor;

import javax.persistence.Entity;
import javax.persistence.GeneratedValue;
import javax.persistence.GenerationType;
import javax.persistence.Id;

@Data
@Entity
@NoArgsConstructor
public class Booking {

    @Id
    @GeneratedValue(strategy = GenerationType.IDENTITY)
    private Integer bookingId;

    private Integer customerId;
    private Integer flightId;
    private String referenceNumber;

}